packages/native-stack/src/views/DebugContainer.native.tsx

```diff
@@ -15,22 +18,26 @@ type ContainerProps = ViewProps & {
  * for detailed explanation.
  */
 export let DebugContainer = (props: ContainerProps) => {
-  return <View {...props} collapsable={false} />;
+  return <ScreenContentWrapper {...props} />;
 };
 
 if (process.env.NODE_ENV !== 'production') {
   DebugContainer = (props: ContainerProps) => {
     const { stackPresentation, ...rest } = props;
 
-    if (Platform.OS === 'ios' && stackPresentation !== 'push') {
+    if (
+      Platform.OS === 'ios' &&
+      stackPresentation !== 'push' &&
+      stackPresentation !== 'formSheet'
+    ) {
       // This is necessary for LogBox
       return (
         <AppContainer>
-          <View {...rest} collapsable={false} />
+          <ScreenContentWrapper {...rest} />
         </AppContainer>
       );
     }
 
-    return <View {...rest} collapsable={false} />;
+    return <ScreenContentWrapper {...rest} />;
   };
 }
```

Comment:
Is there a reason this code is not inside of `react-native-screens`? Seems problematic to leave this upto the consumer.

Since there's going to be major version of screens, I think this is a good time to move this code into screens.

Reply:
Let's do it in separate PR. I'll create a task on board for this. Is that ok?

Reply:
Added this to navigation's board.